src/navigator/Headers.tsx

```diff
@@ -244,10 +262,12 @@ export function HeaderTitleWithTokenBalance({
 
 export function HeaderTitleWithSubtitle({
   title,
+  titleStyle,
```

Comment:
wonder if we should just update this universally instead of a prop. Maybe worth checking with design?

Reply:
Made it an optional prop to avoid impact on any of the existing screens. Currently this is used by the `useScrollAwareHeader.tsx`.

Reply:
I was actually proposing changing every screen to the new style. Seems low risk to me, and avoids another style prop which has to be passed down 2 layers deep, and this would be something that is never cleaned up.

Reply:
Changed all the title styles in beb06fe87c70646fed7c43813ae32436c6aaf06d. A quick look through the app didn't reveal any issues with this change on both large and default fonts.